ci-operator/step-registry/openshift-logging/patch-tlssecurityprofile-apiserver/openshift-logging-patch-tlssecurityprofile-apiserver-commands.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -euo pipefail
+
+export KUBECONFIG="${SHARED_DIR}/kubeconfig"
+
+if [ -z "${LOGGING_CUSTOM_TLS_PROFILE_CONFIG:-}" ]; then
+  echo "The tls security profile config is missing"
```

Comment:
LOGGING_CUSTOM_TLS_PROFILE_CONFIG is only needed when LOGGING_TLS_PROFILE_TYPE is custom.